undistro/zora
# UD-1549: upgrade trivy and controller-runtime dependencies

## Description
Upgrade trivy dependency to handle the CVEs, this necessitated an upgrade to controller-runtime to address incompatibility issues

## Linked Issues

## How has this been tested?
- Build images for operator/worker, deploying to local cluster
- Install zora using local images
- Ensure operator runs without error
- Ensure plugins run, and workers run without error (other than local images)
- Ensure reports are processed and available through the dashboard
- Scan operator and worker images to check these vulnerabilities are no longer present

## Checklist
